revisions.js

```diff
@@ -128,6 +175,32 @@ const classMates = [
 
 // Exercise: Reduce the classMates array to a boolean that indicates whether there is an object with the firstName Mojgan is in it.
 
+
+// const checkForName = classMates.reduce((sum, student) => {
+//   if (student.firstName === "Mojgan") {
+//     return true;
+//   };
+//   return sum;
+// }, false);
+// console.log(checkForName);
```

Comment:
Fantastic solution, feel free to show and explain it to your class mates :D The only thing I would change is rename the variables to be more specific to this case.